### ISSUE TITLE
Export the base Point type for easier interoperability with Flow type

### ISSUE THREAD

**Issue by kripod** [indutny/elliptic#102]

**Comment by indutny** [comment]
@kripod is it a note to yourself? The issue is clearly missing description/motivation/etc.

**Comment by kripod** [comment]
No, this is not just a note. In order to be able to easily verify types using FlowType, the base Point type should be exported from the library.
